source/PlatformTracker.hx

```diff
@@ -0,0 +1,36 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import flixel.input.keyboard.FlxKey;
+
+/**
+ * ...
+ * @author Jess Geiger
+ */
+
+ class PlatformTracker extends Player
+ {
+    public var trackedOffsetX:Float;
+    public var baseObj:FlxObject;
+
+    public function new(?obj:FlxObject, ?offsetX:Float = 0)
```

Comment:
/**
     *  Create object and initialize variables based on PlayState declarations
     *  
     *  @param   trackPlayer is used to make the player accessible to the platform class
     */